namui/src/namui/fs/electron/write_vec_u8.rs

```diff
@@ -0,0 +1,32 @@
+use namui_cfg::namui_cfg;
+use wasm_bindgen::{prelude::wasm_bindgen, JsCast, JsValue};
+
+pub enum WriteVecU8Error {
+    DirNotFound(String),
```

Comment:
`write_vec_u8` will not create dir.
If `dir` aren't exist while running `write("dir/file.txt")`, Instead of creating `dir`, It throw DirNotFound error.

Reply:
We can create all directroy recursively via `fs::system_drive::make_dir()`.
But not via `fs::system_drive::write`

Reply:
Can we check that directory is exists or not?

Reply:
Yes, using read_dir()